core/models/application_version.py

```diff
@@ -225,6 +225,8 @@ def create_unique_version(app, version, created_by, created_by_identity):
             logger.warn(
                 "Version %s is taken for Application %s" %
                 (version, app))
+            if not version:
+                version = "1"
             version += ".0"
```

Comment:
My bad - I think I have this change in a local commit that I never PR'd. My abject apologies.